src/wp-admin/includes/image-edit.php

```diff
@@ -391,6 +397,18 @@ function wp_save_image_file( $filename, $image, $mime_type, $post_id ) {
 				return imagepng( $image, $filename );
 			case 'image/gif':
 				return imagegif( $image, $filename );
+			case 'image/webp':
+				if ( function_exists( 'imagewebp' ) ) {
+					/**
+					 * Filters the WebP compression quality for image file saves.
+					 *
+					 * @since 5.8.0
+					 *
+					 * @param int    $quality Quality level between 0 (low) and 100 (high) of the WebP.
```

Comment:
The `$quality` here is incorrectly indented.

Reply:
Will fix.

Reply:
fixed